pkg/proto/http.go

```diff
@@ -0,0 +1,209 @@
+package proto
```

Comment:
This package was moved over, but not deleted in KCM to avoid even more files being pulled into these PRs